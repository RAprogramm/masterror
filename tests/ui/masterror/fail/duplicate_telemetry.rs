use masterror::{AppCode, AppErrorKind, Masterror};

#[derive(Debug, Masterror)]
#[error("oops")]
#[masterror(
    code = AppCode::Internal,
    category = AppErrorKind::Internal,
    telemetry(),
    telemetry()
)]
struct DuplicateTelemetry;

fn main() {}
