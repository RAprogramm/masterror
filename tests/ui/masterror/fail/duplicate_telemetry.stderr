error: duplicate telemetry(...) block
  --> tests/ui/masterror/fail/duplicate_telemetry.rs:13:5
   |
13 |     telemetry()
   |     ^^^^^^^^^

warning: unused imports: `AppCode` and `AppErrorKind`
 --> tests/ui/masterror/fail/duplicate_telemetry.rs:5:17
  |
5 | use masterror::{AppCode, AppErrorKind, Masterror};
  |                 ^^^^^^^  ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
